softlayer/cpi.yml

```diff
@@ -22,34 +22,37 @@
   path: /resource_pools/name=vms/cloud_properties?
   value:
     domain: ((sl_vm_domain))
-    vmNamePrefix: ((sl_vm_name_prefix))
-    ephemeralDiskSize: 100
-    startCpus: 4
-    maxMemory: 8192
-    deployedByBoshcli: true
-    datacenter:
-      name: ((sl_datacenter))
-    hourlyBillingFlag: true
-    networkComponents:
-    - maxSpeed: 100
+    hostname_prefix: ((sl_vm_name_prefix))
+    ephemeral_disk_size: 100
+    cpu: 4
+    memory: 8192
+    deployed_by_boshcli: true
+    datacenter: ((sl_datacenter))
+    hourly_billing_flag: true
+    local_disk_flag: false
+    max_network_speed: 100
 
 - type: replace
   path: /networks/name=default/subnets/0/dns
   value: [8.8.8.8, 10.0.80.11, 10.0.80.12]
 
+- type: replace
+  path: /networks/name=default/subnets/0/cloud_properties/-
```

Comment:
I think this needs to be something like 
```
path: /networks/name=default/subnets/0/cloud_properties?/vlan_ids
value:
- ((sl_vlan_public))
- ((sl_vlan_private))
```
or else it looks for a non existing `cloud_properties` in the bosh.yml

Reply:
Corrected it, thanks for pointing it out.